prometheus-ksonnet/images.libsonnet

```
@@ -8,7 +8,7 @@ local prometheus_images = import 'prometheus/images.libsonnet';
     {
       watch: 'weaveworks/watch:master-5fc29a9',
       kubeStateMetrics: 'k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.1.0',
-      nodeExporter: 'prom/node-exporter:v1.1.2',
+      nodeExporter: 'prom/node-exporter:v1.2.0',
```

I would upgrade this reference to the latest version that we are using, see https://github.com/grafana/jsonnet-libs/pull/717

```suggestion
      nodeExporter: 'prom/node-exporter:v1.3.1',
```